qiskit_experiments/base_experiment.py

```diff
@@ -201,59 +189,106 @@ def circuits(
             *N*-qubit experiment. The circuits mapped to physical qubits
             are obtained via the :meth:`transpiled_circuits` method.
         """
-        # NOTE: Subclasses should override this method with explicit
-        # kwargs for any circuit options rather than use `**circuit_options`.
-        # This allows these options to have default values, and be
-        # documented in the methods docstring for the API docs.
-
-    def transpiled_circuits(
-        self, backend: Optional[Backend] = None, **kwargs
-    ) -> List[QuantumCircuit]:
-        """Return a list of experiment circuits.
+        # NOTE: Subclasses should override this method using the `options`
+        # values for any explicit experiment options that effect circuit
+        # generation
+
+    @classmethod
+    def _default_options(cls) -> Options:
```

Comment:
It look like the other options have names (e.g. transpile, run, etc...). Perhaps also give this a name? E.g. `_default_experiment_options`? Or something make the intent more clear.